esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2691,11 +2691,24 @@ bool CFileSprayEx::onCopy(IEspContext &context, IEspCopy &req, IEspCopyResponse
         StringBuffer destFolder, destTitle, defaultFolder, defaultReplicateFolder;
         StringBuffer srcNodeGroup, destNodeGroup;
         bool bRoxie = false;
+        const char* srcDali = req.getSourceDali();
         const char* destNodeGroupReq = req.getDestGroup();
-        if(!destNodeGroupReq || !*destNodeGroupReq)
+        if(isEmptyString(destNodeGroupReq))
```

Comment:
picky: there was probably a mix before, but worth making formatting consistent within a function you're modifying, e.g. missing space after if here and on new code on line 2700.

Reply:
Will do